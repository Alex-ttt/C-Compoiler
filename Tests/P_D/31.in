struct a{
	struct b{
		struct c{
			char ind;
		} third;
	};
};