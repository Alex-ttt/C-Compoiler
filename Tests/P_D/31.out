  struct a
          struct b
                  struct c
                          ind char
                  third struct c

   ----=====================================================----
